backend/api/resource_reference.proto

```diff
@@ -23,12 +23,14 @@ enum ResourceType {
   JOB = 2;
   PIPELINE = 3;
   PIPELINE_VERSION = 4;
+  NAMESPACE = 5;
 }
 
 enum Relationship {
   UNKNOWN_RELATIONSHIP = 0;
   OWNER = 1;
   CREATOR = 2;
+  BELONGING = 3;
```

Comment:
Does it make sense to model namespace as the owner of the other resources?
The owner can potentially trigger the cascade delete in delete API call.

Reply:
Sure. I can use the OWNER relationship.